modules/redis/redis.c

```diff
@@ -135,6 +145,32 @@ redis_dd_format_persist_name(const LogPipe *s)
 }
 
 static gboolean
+sendRedisCommand(RedisDriver *self, const char *format, ...)
+{
+  va_list ap;
+  va_start(ap, format);
+  redisReply *reply = redisvCommand(self->c, format, ap);
+  va_end(ap);
+
+  gboolean retval = reply && (reply->type != REDIS_REPLY_ERROR);
+  if (reply)
+    freeReplyObject(reply);
+  return retval;
+}
+
+static gboolean
+test_connection_to_redis(RedisDriver *self)
```

Comment:
Could you rename `test_*` functions to `check_*`? It is confusing to see a function starting with `test_` outside the test code.